app/models/fact_check.rb

```diff
@@ -189,4 +190,45 @@ def detach_claim_if_trashed
       cd.save!
     end
   end
+
+  def set_project_media
+    begin
+      self.create_project_media_for_fact_check
+    rescue RuntimeError => e
+      if e.message.include?("\"code\":#{LapisConstants::ErrorCodes::const_get('DUPLICATED')}") && self.publish_report
+        existing_pm = ProjectMedia.find(JSON.parse(e.message)['data']['id'])
+        if existing_pm.fact_check.language != self.language
+          self.create_project_media_for_fact_check(true)
+        end
+      else
+        # Skip report update as ProjectMedia creation failed and log the failure
+        self.skip_report_update = true
+        Rails.logger.info "[FactCheck] Exception when creating ProjectMedia from FactCheck[#{self.id}]: #{e.message}"
+        CheckSentry.notify(e, fact_check: self.id, claim_description: self.claim_description.id)
+      end
+    end
+  end
+
+  def create_project_media_for_fact_check(is_duplicate = false)
+    pm = ProjectMedia.new
+    if is_duplicate
+      pm.media = Blank.create!
+    else
+      pm.set_original_claim = self.set_original_claim
+    end
+    pm.claim_description = self.claim_description
+    pm.set_status = self.rating
+    pm.set_tags = self.tags
+    pm.skip_check_ability = true
+    pm.save!
```

Comment:
Where are you setting the team?

Reply:
Team value set using ProjectMedia callbacks